charts/jxboot-helmfile-resources/templates/jx-gcactivities-cronjob.yaml

```diff
@@ -15,8 +15,11 @@ spec:
           labels:
             app: gcactivities
             release: jxboot-helmfile-resources
-{{- if .Values.gc.activities.annotations }}
           annotations:
+{{- if .Values.istio.enabled }}
+            sidecar.istio.io/inject: "false"
+{{- end }}
+{{- if .Values.gc.activities.annotations }}
```

Comment:
If annotations are always going to be set, do we need this if loop?

Reply:
Not really. I left it as is to keep changes down to a minimum

Reply:
@ankitm123 Do you want this to be removed?